packages/components/modules/messages/hooks/useNameAndAvatar.ts

```diff
@@ -0,0 +1,40 @@
+import { useCurrentProfile } from '@baseapp-frontend/authentication'
+
+import { useFragment } from 'react-relay'
+
+import { ChatRoomHeaderFragment$key } from '../../../__generated__/ChatRoomHeaderFragment.graphql'
+import { ChatRoomHeaderFragment } from '../graphql/queries/ChatRoomHeaderFragment'
+import { getParticipantCount, isGroupChat } from './utils'
+
+const useNameAndAvatar = (roomHeaderRef: ChatRoomHeaderFragment$key) => {
```

Comment:
This is written in a reusable way and should also be used to obtain the chat name and avatar in the chat list. I will update this when the corresponding PRs are merged.